bip-0353.mediawiki

```diff
@@ -154,13 +154,41 @@ vvjqj96p8dz2h" "rsh5z5n27qfk6svjz5pmkh0smq26k0j2j4q36xgq3r5qzet9kuhq4lydpen5mskx
+
+This encodes address <tt>BC1QYYQ734DCNHHCWYW8YJXSWXG4P7K9NXU0QT84QZ</tt> to <tt>override.x_domain_cname_wild@dnssec_proof_tests.bitcoin.ninja</tt> valid on August 6, 2025.
```

Comment:
Decoding the hex gave me a slightly different address, but it might be an error on my part. Can you double-check?